azure-devops/azext_devops/dev/common/credential_store.py

```diff
@@ -54,11 +56,12 @@ def get_password(self, key):
 
         try:
             return keyring.get_password(key, self._USERNAME)
```

Comment:
may be a corner case, but what if set has issues and get does not have issue.
With this change should we start looking in both the places for password?